systemtest/src/test/java/io/strimzi/systemtest/metrics/PrometheusST.java

```diff
@@ -0,0 +1,91 @@
+        assertThat("Prometheus service not found", kubeClient().getService(PROMETHEUS) != null);
+        assertThat("Prometheus service port is not 9090", kubeClient().getService(PROMETHEUS).getSpec().getPorts().get(0).getPort() == 9090);
+    }
+
+    @Test
+    public void testAlertManagerService() {
+        assertThat("AlertManager service not found", kubeClient().getService(ALERTMANAGER) != null);
+        assertThat("AlertManager service port is not 9090", kubeClient().getService(ALERTMANAGER).getSpec().getPorts().get(0).getPort() == 9093);
+    }
+
+    @Test
+    public void testAlertManagerPodIsUp() {
+        assertThat("AlertManager pod not found", kubeClient().getPod(ALERTMANAGER_POD) != null);
+        int conditionsSize = kubeClient().getPod(ALERTMANAGER_POD).getStatus().getConditions().get(0).getStatus().length();
+        assertThat("AlertManager pod is not ready", kubeClient().getPod(ALERTMANAGER_POD).getStatus().getConditions().get(conditionsSize - 1).getStatus().equals("True"));
+    }
+
+    @Test
+    public void testPrometheusPodIsUp() {
+        assertThat("Prometheus pod not found", kubeClient().getPod(PROMETHEUS_POD) != null);
+        int conditionsSize = kubeClient().getPod(PROMETHEUS_POD).getStatus().getConditions().get(0).getStatus().length();
+        assertThat("Prometheus pod is not ready", kubeClient().getPod(PROMETHEUS_POD).getStatus().getConditions().get(conditionsSize - 1).getStatus().equals("True"));
+    }
+
+    @Test
+    public void testSecretsCreated() {
+        assertThat("additional-scrape-configs secret not found", kubeClient().getSecret("additional-scrape-configs") != null);
+        assertThat("additional-scrape-configs secret does not contain key prometheus-additional.yaml", kubeClient().getSecret("additional-scrape-configs").getData().get("prometheus-additional.yaml") != null);
+        assertThat("alertmanager-alertmanager secret", kubeClient().getSecret("alertmanager-alertmanager") != null);
+        assertThat("alertmanager-alertmanager secret does not contain key alertmanager.yaml", kubeClient().getSecret("alertmanager-alertmanager").getData().get("alertmanager.yaml") != null);
+    }
+
+    // No need to recreate environment after failed test. Only values from collected metrics are checked
+    @Override
+    protected void recreateTestEnv(String coNamespace, List<String> bindingsNamespaces) { }
+
+    @BeforeAll
+    void setupEnvironment() {
+        LOGGER.info("Creating resources before the test class");
+        prepareEnvForOperator(NAMESPACE);
+
+        createTestClassResources();
+        cmdKubeClient().apply(StUtils.downloadYamlAndReplaceNameSpace("https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml", NAMESPACE));
+
+        cmdKubeClient().exec("oc", "create", "secret", "generic", "additional-scrape-configs", "--from-file=prometheus-additional.yaml=../metrics/examples/prometheus/additional-properties/prometheus-additional.yaml", "-n", NAMESPACE);
```

Comment:
Any specific reason this is using `oc`? Seems to be a generic command which can use `kubectl`.

Reply:
Ye, for that it will be better to create new method in `BaseCmdKubeClient` which will create secret and instead of using `oc` you can use `cmd()` method, which return `oc` or `kubectl` based on used cluster